core/store/src/trie/mod.rs

```diff
@@ -58,7 +58,7 @@ pub mod outgoing_metadata;
 mod prefetching_trie_storage;
 mod raw_node;
 pub mod receipts_column_helper;
-pub mod resharding_v2;
+
```

Comment:
Remove new line. When removing imports, always remove the new line as well. Add this to knowledge.